corehq/apps/commtrack/sms.py

```diff
@@ -410,7 +410,7 @@ def process_transactions(E, transactions):
         else:
             transfers.append(tx)
 
-    return process_balances(E, balances), process_transfers(E, transfers)
+    return process_transfers(E, transfers), process_balances(E, balances)
```

Comment:
I think the easier change here would have just been to switch the order in

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/commtrack/sms.py#L477

now it's pretty confusing because `balances` and `transfers` are flipped in that function from what they really are.

can you clean this up?